ci/tests/backend/test_alba.py

```diff
@@ -222,3 +272,40 @@ def ovs_3490_add_remove_preset_test():
     status, message = alba.remove_preset(alba_backend, name)
     assert status, "Remove preset failed with: {0}".format(message)
     assert not is_preset_present(name), "Preset with name {0} is not present".format(name)
+
+
+def ovs_2263_verify_alba_namespace_cleanup_test():
+    """
+    {0}
+    """.format(general.get_function_name())
+
+    general.check_prereqs(testcase_number=9,
+                          tests_to_run=testsToRun)
+    no_namespaces = 3
+
+    backend = generic.get_backend_by_name_and_type(BACKEND_NAME, BACKEND_TYPE)
+    if not backend:
+        raise SkipTest()
+
+    alba_backend = alba.get_alba_backend(backend['alba_backend_guid'])
+    if not alba_backend:
+        raise SkipTest()
+
+    for nmspc_index in range(no_namespaces):
+        alba.create_namespace(BACKEND_NAME, 'nmspc_{0}'.format(nmspc_index), 'default')
+    result = alba.list_namespaces(BACKEND_NAME)
+    assert len(result) == no_namespaces, "Expected {0} namespaces present on the {1} backend, found {2}".format(no_namespaces, BACKEND_NAME, len(result))
+    vpool_generic.add_generic_vpool()
+    vmachine_test.check_template_exists()
+    for disk_index in range(no_namespaces):
+        vmachine_test.create_raw_vdisk_from_template('/var/tmp/templates/','debian.qcow2','jessica','disk{0}'.format(disk_index))
+    result = alba.list_namespaces(BACKEND_NAME)
+    assert len(result) == 2 * no_namespaces + 1, "Expected {0} namespaces present on the {1} backend, found {2}".format(2 * no_namespaces + 1, BACKEND_NAME, len(result))
+    _, __ = general.execute_command("rm -rf /mnt/{0}/*.raw".format('jessica'))
+    general.api_remove_vpool('jessica')
+    result = alba.list_namespaces(BACKEND_NAME)
+    assert len(result) == no_namespaces, "Expected {0} namespaces present on the {1} backend, found {2}".format(no_namespaces, BACKEND_NAME, len(result))
+    for namespace in result:
+        alba.delete_namespace(BACKEND_NAME, namespace['name'])
+    result = alba.list_namespaces(BACKEND_NAME)
+    assert len(result) == 0, "Expected no namespaces present on the {1} backend, found {2}".format(no_namespaces, BACKEND_NAME, len(result))
```

Comment:
it might be more interesting to move this
https://github.com/openvstorage/integrationtests/pull/109/files
to the vpool test
as you need a vpool for this anyway
no need to add creation of a vmachine
vdisk / truncate of a file should be sufficient